release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -182,19 +193,25 @@ Find .5-1 hour of time to meet with shadows and explain the release process. Wal
 
 ### Ensure access is set up
 
-1. Make sure all shadows have edit access to the enhancement spreadsheet.
+1. Add shadows to the "official" release team file, e.g.: [1.21 release](https://github.com/kubernetes/sig-release/pull/1432)
+
+1. As a lead, make sure you are part of the [sig-docs-en-owners](https://github.com/orgs/kubernetes/teams/sig-docs-en-owners), e.g: [1.21 release](https://github.com/kubernetes/website/pull/26114).
+
+1. Add docs team to the `website-milestone-maintainers` group in k/org/config/kubernetes/sig-docs/teams.yaml and `release-team` group in k/org/kubernetes/sig-release/teams.yaml, e.g: [1.21 release](https://github.com/kubernetes/org/pull/2445).
 
-1. As a lead, make sure you are part of the [milestone-maintainers](https://github.com/orgs/kubernetes/teams/milestone-maintainers) and [sig-docs-en-owners](https://github.com/orgs/kubernetes/teams/sig-docs-en-owners).
+1. Make sure all shadows have edit access to the enhancement spreadsheet. Contact the enhancements lead to grant access for you and shadows.
 
 1. You need push access to the Kubernetes website repo (contact a SIG Docs chair if you don't have it)
 
-   ⚠️ (**This should be done by the Docs Lead**) Open the [integration branch] by creating a pull request against `master` referencing the `dev-[future release]` branch e.g., the [Release 1.14](https://github.com/kubernetes/website/pull/13174) PR uses the branch `dev-1.14`.
+   ⚠️ (**This should be done by the Docs Lead**) Open the [integration branch] by creating a pull request through the GitHub UI against `master` referencing the `dev-[future release]` branch e.g., the [Release 1.14](https://github.com/kubernetes/website/pull/13174) PR uses the branch `dev-1.14`.
```

Comment:
since the examples above point to v1.21, would it make sense to update all of them to match, instead of point to different examples from different releases?

Reply:
I don't think that matters too much since the way the PR should be made has not changed. If the process has changed, I believe the example PR also changed with it to ensure that latest examples are listed in the doc.

Reply:
Updated examples to 1.21